hudi-utilities/pom.xml

```diff
@@ -85,7 +85,6 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
-      <scope>test</scope>
```

Comment:
Scope is getting broadened. Can you shade it in all the bundles? We're not sure which other dependency will pull this transitively, or maybe user is adding hudi-utilities-bundle as a dependency in their project which also has probobuf-java-util and there could be some conflict.

Reply:
Should we be shading protobuf as well?

Reply:
Updated to shade `com.google.protobuf` in the bundles

Reply:
@the-other-tim-brown @codope let's take shading up separately

First of all i don't see us even including "protobuf-java-util" in our bundles, we're just adding it as compile dependency, therefore there's no need to be tackling shading in this PR

Reply:
Should I revert that last commit then?

Reply:
@alexeykudinkin and @codope

Reply:
Yes, let's keep changes bounded -- shading is the right change but there's no reason to take it in this PR, since we're adding Proto deps to bundles